Automattic/wp-calypso
# Import: fix other links not redirecting to correct location

#### Proposed Changes

* Adjusted getFinalImporterUrl() URL generation logic to build the link for other importers
* Redirect the `navigation.submit` requests that are triggered by clicking other importers' `Import your content` button to `exitFlow` because the other importer pages are out of the Stepper framework

#### Testing Instructions

* Go to `/setup/importList?siteSlug={SITE_SLUG}`
* Click `Blogroll` link to proceed to `setup/importReady?siteSlug={SITE_SLUG}`
* Check if `Import your content` leads to `https://{SITE_SLUG}/wp-admin/admin.php?import=opml`
* Go to `/setup/importList?siteSlug={SITE_SLUG}`
* Click `Ghost` link to proceed to `setup/importReady?siteSlug={SITE_SLUG}`
* Check if `Import your content` leads to `https://{SITE_SLUG}/wp-admin/admin.php?import=ghost_import`
* Go to `/setup/importList?siteSlug={SITE_SLUG}`
* Click `Tumblr` link to proceed to `setup/importReady?siteSlug={SITE_SLUG}`
* Check if `Import your content` leads to `https://{SITE_SLUG}/wp-admin/admin.php?import=tumblr`
* Go to `/setup/importList?siteSlug={SITE_SLUG}`
* Click `LiveJournal` link to proceed to `setup/importReady?siteSlug={SITE_SLUG}`
* Check if `Import your content` leads to `https://{SITE_SLUG}/wp-admin/admin.php?import=livejournal`
* Go to `/setup/importList?siteSlug={SITE_SLUG}`
* Click `Movable Type & TypePad` link to proceed to `setup/importReady?siteSlug={SITE_SLUG}`
* Check if `Import your content` leads to `https://{SITE_SLUG}/wp-admin/admin.php?import=mt`
* Go to `/setup/importList?siteSlug={SITE_SLUG}`
* Click `Xanga` link to proceed to `setup/importReady?siteSlug={SITE_SLUG}`
* Check if `Import your content` leads to `https://{SITE_SLUG}/wp-admin/admin.php?import=xanga-wxr`

Related to https://github.com/Automattic/wp-calypso/issues/64277
